index.js

```diff
@@ -28,7 +28,15 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let allTitles = [];
+  for (let i = 0; i < movies.length; i++) {
```

Comment:
We should maybe be using a `for...of` loop here since we aren't using `i` for anything outside of accessing elements in the `movies` array.